csm/common/ha/cluster_management/operations.py

```diff
@@ -179,3 +181,56 @@ def execute(self, cluster_manager, **kwargs):
             err_msg = f"{const.CLUSTER_OPERATIONS_ERR_MSG} : {e}"
             Log.error(err_msg)
 
+class NodeMarkFailure(Operation):
+    """
+    Process mark node failure request.
+    """
+    def validate_arguments(self, **kwargs):
```

Comment:
No need to validate. CSM does not validate semantics. Pass this method

Reply:
Addressed